examples/dtls/pki/cert_gen.go

```diff
@@ -44,6 +44,9 @@ func GenerateCA() (ca *x509.Certificate, cert, key []byte, priv *ecdsa.PrivateKe
 
 	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
 	serialNumber, err := rand.Int(sequentialBytes(128), serialNumberLimit)
+	if err != nil {
+		return
```

Comment:
pls return err

Reply:
It does. Look at the function signature, it returns 4 named parameters, the last one is `err` and `err` is assigned to on line 46.